package.json

```diff
@@ -19,17 +18,17 @@
   "bugs": {
     "url": "https://github.com/seatgeek/react-infinite/issues"
   },
-  "browser": "build/react-infinite.js",
+  "module": "./es",
+  "main": "./es/react-infinite.js",
   "scripts": {
     "test": "jest",
     "lint": "eslint --ext .jsx --ext .js ./src ./__tests__ ./pipeline ./cypress",
     "fix": "eslint --fix --ext .jsx --ext .js ./src ./__tests__ ./pipeline ./cypress",
     "typecheck": "flow version && flow check",
     "verify": "npm test && npm run typecheck && npm run lint",
     "preversion": "npm run verify",
-    "prepublish": "gulp release",
-    "lint-staged": "lint-staged",
-    "precommit": "lint-staged"
+    "build": "npx babel ./src --out-dir ./es --env-name esm",
```

Comment:
This is the crux of all changes in this branch. Upgraded babel, got rid of gulp and other 'build automation' things (all outdated). Instead just use babel cli to quickly build /es/ modules in place of /dist/. This makes sense for our usage of react-infinite as we bundle it into our bigger project(s) and never directly load it into browser like from a CDN.